app/views/users/posts.html.erb

```diff
@@ -6,11 +6,20 @@
 
 <h1>Posts by <%= user_link @user %></h1>
 
-<div class="button-list is-gutterless">
-  <%= link_to 'Score', query_url(sort: 'score'), class: 'button is-muted is-outlined ' + (active_search?('score') ? 'is-active' : ''),
-              role: 'button', 'aria-label': 'Sort by score' %>
-  <%= link_to 'Age', query_url(sort: 'age'), class: 'button is-muted is-outlined ' + (active_search?('created_at') ? 'is-active' : ''),
-              role: 'button', 'aria-label': 'Sort by age' %>
+<div>
+  <% post_count = @posts.count %>
+  <div class="has-color-tertiary-500 category-meta" title="<%= post_count %>">
+    <div>
+      <%= short_number_to_human post_count, precision: 1, significant: false %>
+      <%= 'post'.pluralize(post_count) %> &middot;
+    </div>
```

Comment:
I forgot to mention that the middot (`&middot;`) used to separate the post count from other buttons is redundant here (since there are no other buttons). I won't add a suggested change yet as I don't want this trivial change to mess up the larger suggestion it would clash with.

Once the larger change is applied (if you're happy with it) I'm happy to make a separate suggestion for the middot, or you could just make a separate commit - whatever's easier.

Reply:
I had to edit for the stray div anyway, so I got rid of the `&middot;` while I was there.

I agree that we want a general fix for the misplaced tooltip text and should do it separately.